config/cliff.toml

```diff
@@ -51,6 +51,8 @@ filter_commits = false
 tag_pattern = "v[0-9]*"
 # regex for skipping tags
 skip_tags = "v0.1.0-beta.1"
+# regex for ignoring tags
+ignore_tags = ""
```

Comment:
Can you add this line to other configuration files in `examples/` folder as well?

Reply:
Sure. btw, is it okay to set the value to `""`?
Because of this, I have to check empty here.
https://github.com/kenji-miyake/git-cliff/blob/ee85a9b988f6818cecccbd0ecf19aa143f67b5a9/git-cliff/src/lib.rs#L128-L130

Probably other options are:
- Don't define the variable
- Set to null (but impossible with `toml`?)

Reply:
Done in b8977db2bb59d3eaad5e9b1ddd77cf3053738c62.

Reply:
I think it is fine to keep it empty 👍🏼